src/components/header/NavList/NavList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+interface NavListProps {
+  items: React.ReactNode[]
+  keyPrefix: string
+  ulClass?: string
+  liClass?: string
+}
+
+// Should the UL or LI html attributes be passed in?
+export const NavList = (
+  props: NavListProps & React.HTMLAttributes<HTMLUListElement>
+): React.ReactElement => {
+  const { items, keyPrefix, ulClass, liClass, ...ulProps } = props
+
+  return (
+    <ul className={ulClass} {...ulProps}>
+      {items.map((item, i) => (
+        <li key={`${keyPrefix}_${i}`} className={liClass}>
+          {item}
+        </li>
+      ))}
+    </ul>
+  )
+}
```

Comment:
That works for me. I think all it does is tie the NavList component more heavily to the navigation, which is fine. I'm not sure it makes sense to include the isOpen just for the subnav. I think it can still be passed in through the props. I made the most of the changes and will push them shortly. Let me know what you think.